pkg/clients/fake.go

```diff
@@ -23,6 +23,8 @@ import (
 	fakecore "k8s.io/client-go/kubernetes/fake"
 )
 
+const CSIDriverNamespace = "openshift-cluster-csi-drivers"
```

Comment:
I guess it need not move. It can still be used by other operators. +1

Reply:
I added this as a constant here because it is needed when creating the fake client, for example at line 33 of this file.
It was not possible to get the namespace from the builder as we did it [here](https://github.com/openshift/csi-operator/pull/294/files#diff-2a0b4d2a23052c35008c27b7751abd60f924853f04007197f8eedbaaa25c6c81R49) .
Note that the idea of the PR was to make this constant [here](https://github.com/openshift/csi-operator/pull/294/files#diff-31bbef804f62eb42a038e0b6402d37c03d3989e1cf9a6bc0ac9a8e5a50bdc53bL25) configurable and use the new configured namespace that was propagated in the client.

Reply:
I was asking why it moved from `pkg/clients/clients.go` to `pkg/clients/fake.go` as that seems unnecessary when they are in the same package. Thinking on it more, I assume you did this because nothing outside of this file should be using it now and they should instead be using `Clients.GuestNamespace`. If so, you've replaced most references but you still have some:

```
$ gh pr checkout 294

$ ag CSIDriverNamespace
pkg/openstack-cinder/config/configsync.go
220:                    Namespace: clients.CSIDriverNamespace,

pkg/clients/fake.go
26:const CSIDriverNamespace = "openshift-cluster-csi-drivers"
33:     controlPlaneKubeInformers := v1helpers.NewKubeInformersForNamespaces(controlPlaneKubeClient, controllerNamespace, "", CSIDriverNamespace)
49:     guestKubeInformers := v1helpers.NewKubeInformersForNamespaces(guestKubeClient, controllerNamespace, "", CSIDriverNamespace)
67:             Namespace:  CSIDriverNamespace,
79:             EventRecorder: events.NewKubeRecorder(guestKubeClient.CoreV1().Events(CSIDriverNamespace), "fake", &fakeObjectRef),

pkg/driver/openstack-cinder/openstack_cinder.go
161:            clients.CSIDriverNamespace,
163:            c.GetConfigMapInformer(clients.CSIDriverNamespace),
166:            c.GetConfigMapInformer(clients.CSIDriverNamespace).Informer(),
```

I'm thinking this is a mistake and you missed these? If so, you probably want to fix this. Also, I think this attribute should probably be lower case now (private) to prevent this happening in the future?

```suggestion
const csiDriverNamespace = "openshift-cluster-csi-drivers"
```

Reply:
@stephenfin yeah, that was the point:

> nothing outside of this file should be using it now

Thanks for highlighting the places I missed in cinder, that happened after the last rebase. Will fix it.